hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/HBaseInterClusterReplicationEndpoint.java

```diff
@@ -450,13 +447,6 @@ protected void doStop() {
     notifyStopped();
   }
 
-  // is this needed? Nobody else will call doStop() otherwise
-  @Override
-  public State stopAndWait() {
```

Comment:
Guava's Service interface has changed. stopAndWait() is replaced with stopAsync() and awaitTerminated(). 

In any case, I reviewed Guava code and doStop is in fact called when the service is transitioned out of RUNNING state by stopAsync. Something like this override is not needed.